pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/PersistentTopicsTest.java

```diff
@@ -200,6 +203,76 @@ public void testGetSubscriptions() {
         Assert.assertEquals(responseCaptor.getValue().getStatus(), Response.Status.NO_CONTENT.getStatusCode());
     }
 
+    @Test
+    public void testTerminatePartitionedTopic() {
+        String testLocalTopicName = "topic-not-found";
```

Comment:
Seems most of the test content is duplicate with above `testGetSubscriptions()`,  Could we refine this test case to focus on topic Terminate?

Reply:
Simplified only for termination of topic.